src/components/Notification/Notification.styles.tsx

```diff
@@ -0,0 +1,54 @@
+import color from "../../styles/colors";
+import fonts from "../../styles/fonts";
+import resetCSS from "../../styles/reset";
+
+const initialStyles = `
+    ${resetCSS}
+    ${fonts.text}
+    background: #112F5C;
+    border-radius: 20px;
+    color: white;
+    min-width: 270px;
+    max-width: 25%;
+    height: 84px;
+    padding: 16px;
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+`
+
+const box = `
+    display: grid;
+    width: 100%;
+    margin-left: 0.5rem;
```

Comment:
is REM needed, i would rather not have a mix of values, lets try to stick to `px` and `%` unless you need this?

Reply:
will change to px